Formula/crosstool-ng.rb

```diff
@@ -3,7 +3,7 @@ class CrosstoolNg < Formula
   homepage "https://crosstool-ng.github.io/"
   url "http://crosstool-ng.org/download/crosstool-ng/crosstool-ng-1.24.0.tar.xz"
   sha256 "804ced838ea7fe3fac1e82f0061269de940c82b05d0de672e7d424af98f22d2d"
-  license "LGPL-2.1"
+  license "GPL-2.0-only"
   revision 1
```

Comment:
```suggestion
  revision 2
```
Sorry, I forgot this. We need to bump the revision so that users who have this installed get the updated version of the formula.

Reply:
yeah, that's correct. I forgot that too, thanks!